Tools/baselineDef.cc

```diff
@@ -119,6 +141,75 @@ bool BaselineVessel::SetupTopTagger(std::string CfgFile_)
   return true;
 }       // -----  end of function BaselineVessel::SetupTopTagger  -----
 
+// ===  FUNCTION  ============================================================
+//         Name:  BaselineVessel::prepareDeepTopTagger
+//  Description:  
+// ===========================================================================
+void BaselineVessel::prepareDeepTopTagger()
+{
+  jetsLVec_forTagger      = new std::vector<TLorentzVector>();
+  recoJetsBtag_forTagger  = new std::vector<float>();
+  qgLikelihood_forTagger  = new std::vector<float>();
+  *jetsLVec_forTagger     = tr->getVec<TLorentzVector>(jetVecLabel);
+  *recoJetsBtag_forTagger = tr->getVec<float>(CSVVecLabel);
+  *qgLikelihood_forTagger = tr->getVec<float>(qgLikehoodLabel);
+
+  tr->registerDerivedVec("jetsLVec_forTagger" + firstSpec, jetsLVec_forTagger);
```

Comment:
Is this necessary, we are just making deep copies for fun at this point?

Reply:
It is more for backward comparability. We used to store the jets used in the top tagger. But this might need be needed anymore. Let us see whether people are using these, otherwise we can remove them